spec/takeaway_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { double :menu }
+  let(:order) { double :order }
+  let(:sms_sender) { double :smssender }
+  subject(:takeaway) { described_class.new(menu, order, sms_sender) }
+ 
+  it 'contains a @menu that is an instance of Menu' do
+    ta = described_class.new()
```

Comment:
I think that you chose not to use a double here because you are trying to test the default behaviour, is that correct?  Perhaps the thing to test is that whatever get's passed in as the first arg is assigned to @menu (if you want to test that assignment).  So you could test that `takeaway.menu` is equal to `menu`.

Having said that, people tend not to test the assignment of instance vars like this.  There will be some behaviour somewhere that relies on these vars, which will also test for their correct assignment.  It's worth doing a tiny bit of reading about this though as you're bound to find alternative opinions!